envoy/auth_server/.env.example

```diff
@@ -8,3 +8,8 @@ GRPC_HOST_PORT=path_auth_data_server:50051
 # Default is "false" if not set.
```

Comment:
Do you mind adding a couple Makefile targets to help test a request with the URL & the header?

For reference, something similar to this: https://github.com/pokt-network/poktroll/blob/938f8183b9ab1fd1f6b1bc5d8ff2bba7ab6e2a18/makefiles/relay.mk#L7

Reply:
Added to Makefile and referenced in docs.